### ISSUE TITLE
Customizing for all resources doesn't work

### ISSUE THREAD

**Issue by mishaker** [thoughtbot/administrate#598]
When I do a 
`rails generate administrate:views`
I have all generated templates. but when I change them nothing happens. 
But when I customize for a specific resource: 
`rails generate administrate:views User`
Then it works !

**Comment by carlosramireziii** [comment]
Hey @misaghshakeri - There's a new release that merges #747 that may solve your problem. Would you be able to check to see if this is still an issue with the latest version. Thanks!

**Comment by gfauredumont** [comment]
Hi,
I'm having the same issue when generating the layout (and it's components).
(easy) workaround is to set the layout explicitly in `<adm_namespace>/application_controller.rb` like so:
```ruby
layout 'admin/application'
```
Then you can use the overrided partials with their full path:
```ruby
<%= render "admin/application/sidebar" -%>
```

I'm not completely sure how I could help on Administrate (do you need any newby help ?!!), but I sure love it: I no longer have to fight against AA....

THANK YOU GUYS !!!

**Comment by jtannas** [comment]
I ran into this and think I can narrow down the issue.
After running `rails generate administrate:views:navigation`, the edits were not taking effect.
Once I renamed `app/views/admin/application/_navigation.html.erb` to `app/views/administrate/application/_navigation.html.erb` and restarted the server then changes were applied as expected. Seems like some piece of code is looking at the incorrect path.
